tracing/README.md

```diff
@@ -0,0 +1,20 @@
+# Tracing
+
+Currently this `tracing` module supports two different tracing abstractions
+ * OpenTracing
+ * OpenTelemetry 
+ 
+where both use `Jaeger` as the actual tracing implementation.
+
+Actual tracing usage is triggered by specific `service name` env var usage.
+
+List of env vars we need to set:
+
+#### OpenTracing
+ * JAEGER_SERVICE_NAME -- this triggers OpenTracing tracing
+ * JAEGER_AGENT_HOST -- host name where the Jaeger traces are sent
+
+#### OpenTelemetry
+ * OTEL_SERVICE_NAME -- this triggers OpenTelemetry tracing
+ * OTEL_EXPORTER_JAEGER_ENDPOINT -- url where the Jaeger traces are sent
```

Comment:
Since you expect both to feed into Jaeger ... does the expected `JAEGER_AGENT_HOST` differ from `OTEL_EXPORTER_JAEGER_ENDPOINT`? Or is it expected to be the same?

Reply:
I think they differ.
At least in the Strimzi CO test they were different.